intern.json

```diff
@@ -4,11 +4,17 @@
 		"name": "@dojo/interop"
 	},
 	"browser": {
+		"suites+": [
+			"./dist/dev/tests/unit/dgrid/all.js"
```

Comment:
Don't think we should need to add this

Reply:
The dgrid wrapper tests don't work in node.

Reply:
Gotcha. Shame.